clang/test/SemaSYCL/streams.cpp

```diff
@@ -0,0 +1,1273 @@
+// RUN: %clang_cc1 -S -I %S/Inputs -fsycl -fsycl-is-device -triple spir64 -ast-dump %s | FileCheck %s
+
+#include <sycl.hpp>
```

Comment:
It looks like other tests use following pattern:
```c++
#include "Inputs/header.hpp"
```
instead of 
```c++
// RUN: %clang_cc1 -I %S/Inputs ...
#include "header.hpp"
```
I suggest we refactor our SYCL LIT tests to follow the patter w/o additional compiler options.
I can create a separate PR, if there are no objections.

Reply:
I don't see a problem with it.  Getting them all the same would be nice.  In this case, I ended up copy/pasting the cc1 invocation/include from another file, so there's definitely more like it that have the same problem.